src/main/webapp/event-info-page/eventInfo.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyB0s69v5OFVxuH0cN5IS96B3jwhnjB7tTo"></script>
```

Comment:
Could you replace this with the API key that Mingyi generated? (It's for the team so we can all use it.)
https://maps.googleapis.com/maps/api/js?key=AIzaSyDlz0NQ1mc6_UU6OHExGKIrpnkJ5q9BqPs

Reply:
Sure I will try it!